packages/okta-react-native/README.md

```diff
@@ -50,145 +40,404 @@ After you have created the application there are two more values you will need t
+
+Perform the following Manual installation steps if you're not using `react-native link`.
+
+#### Manual installation (Optional)
+
+##### iOS
+
+1. In XCode, in the project navigator, right click `Libraries` ➜ `Add Files to [your project's name]`
+2. Go to `node_modules` ➜ `@okta/okta-react-native` and add `ReactNativeOktaSdkBridge.xcodeproj`
+3. In XCode, in the project navigator, select your project. Add `libReactNativeOktaSdkBridge.a` to your project's `Build Phases` ➜ `Link Binary With Libraries`
+4. Run your project (`Cmd+R`)<
+
+##### Android
+
+1. Open up `android/app/src/main/java/[...]/MainApplication.java`
+  - Add `import com.oktareactnative.OktaSdkBridgePackage;` to the imports at the top of the file
+  - Add `new OktaSdkBridgePackage()` to the list returned by the `getPackages()` method
+2. Append the following lines to `android/settings.gradle`:
+    ```
+    include ':@okta/okta-react-native'
+    project(':@okta/okta-react-native').projectDir = new File(rootProject.projectDir, '../node_modules/@okta/okta-react-native/android')
+    ```
+3. Insert the following lines inside the dependencies block in `android/app/build.gradle`:
+    ```
+      compile project(':@okta_okta-react-native')
+    ```
+
+### iOS Setup
+
+To setup iOS, there are three steps that you must take.
+
+1. [Make sure your iOS app's deployment target is `11.0` and above.](#set-ios-deployment-target)
+2. [Install Okta Open ID Connect iOS.](#install-okta-open-id-connect-ios)
+3. [Make sure you also configure Swift.](#swift-configuration)
+
+#### Set iOS Deployment Target
+
+This library supports iOS version `11.0` and above. Go to your project -> `Build settings` -> `iOS Deployment Target`, and set it to at least version `11.0`.  
+
+#### Install Okta Open ID Connect iOS
+This library depends on the native [Okta OIDC iOS](https://github.com/okta/okta-oidc-ios) library. This library is not distributed as part of the React Native library to keep your dependency management consistent. 
+
+There are two ways to add Okta OIDC iOS to your dependencies:
+
+1. **CocoaPods**
+   With [CocoaPods](https://guides.cocoapods.org/using/getting-started.html), make sure your `Podfile` looks like this:
+
+    ```   
+    platform :ios, '11.0'
+
+    target 'Test9OldVersion' do
+      use_frameworks!
+
+      pod 'OktaOidc', '~> 3.0'
+
+    end
+    ```
+
+   Then run `pod install`.
+
+<!-- 2. **Carthage**
+   To integrate this SDK into your Xcode project using [Carthage](https://github.com/Carthage/Carthage), specify it in your `Cartfile`:
+
+    ``` 
+    github "okta/okta-oidc-ios"
+    ```
+
+   Then run `carthage update --platform iOS`.
+
+   Drag and drop `OktaOidc.framework` from `ios/Carthage/Build/iOS` to `Frameworks` in Xcode.
+
+   Add a copy files build step for `OktaOidc.framework`: open Build Phases on Xcode, add a new "Cope Files" phase, choose "Frameworks" as destination, add `OktaOidc.framework` and ensure "Code Sign on Copy" is checked. -->
+
+2. **Static Library**
```

Comment:
I would delete this section. It causes a lot of confusion:
- we don't add any static libraries here
- this type of configuration requires some extra steps: git checkout of oidc repo with particular tag, then adding it as a dependency

I think pod installation should be enough for now. In next release we will add Carthage support